README.md

```diff
@@ -115,4 +115,13 @@ Please create a GitHub issue to report an issue or ask questions.
 ## Contributing
 
 Contributions to the OpenCL ICD Loader are welcomed and encouraged.
-You will be prompted with a one-time "click-through" CLA dialog as part of submitting your pull request or other contribution to GitHub.
\ No newline at end of file
+You will be prompted with a one-time "click-through" CLA dialog as part of submitting your pull request or other contribution to GitHub.
+
+## Table of Debug Environment Variables
```

Comment:
Is there a reason to call these as "debug" variables?

Reply:
The Vulkan loader calls them "debug environment variables":

https://github.com/KhronosGroup/Vulkan-Loader/blob/master/loader/LoaderAndLayerInterface.md#table-of-debug-environment-variables

Other ICD loaders simply call them "environment variables".

I have a slight preference for "debug environment variables", since they probably won't need to be set (and shouldn't be set!) for most end-user installations, but I don't have a very strong opinion, and I'd be fine changing to just "Table of Supported Environment Variables".

Reply:
I agree with the rationale.